.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+import 'cypress-real-events/support';
+
+it('allows creating a company', () => {
+  cy.login();
+  cy.get('[title="Home Menu"]').click();
+  cy.contains('Settings').click();
+  cy.contains('Users & Companies').click();
+  cy.get('[data-menu-xmlid="base.menu_action_res_company_form"]').click();
+  cy.get('.o_list_button_add').click();
```

Comment:
The creation buttons are usually easily identifiable by their text (they are not really many of them on screen). Can we replace this with `cy.contains` instead?

Reply:
I did!